src/styles/colors.js

```diff
@@ -9,3 +9,18 @@ export const PRIMARY_WEAK = '#73dbff';
 export const PRIMARY_STRONG = '#17bcde';
 
 export const GREEN = '#b5ce4d';
+
+export const ARTICLES = {
```

Comment:
Hm... I'm not so happy about that. If we only use the colors for the articles, we can instead move that object to `ChooseArticle`. If they are used somewhere else, I would rather rename them to something more general, e.g.
```.js
const SELECTION_1_STRONG = '#0000fe'
const SELECTION_1_WEAK = '#6666fe'
```
and build that object in `ChooseArticle` if needed.